coralproject/talk
# Post comment

## What does this PR do?

Enables comment and reply posting with the new backend


## How do I test this PR?

1. `npm run build`
2. `npm start`
3. Visit `http://localhost:3000/client/coral-embed-stream/samplearticle.html`
4. You should be able to post comment and replies.
5. If you reload the page the comments and replies should persist.
6. Other stuff (comment count, etc) should work as expected.

@coralproject/tech
